dist/README-dist.md

```diff
@@ -0,0 +1,77 @@
+# Distribution-specific builds
+
+Maintainer notes for building and publishing various "finished product"
+produced by or with the Unum SDK.
+
+- [OpenWrt (.ipk and full images)][2]
```

Comment:
If we are dropping openwrt_generic, so no more .ipk
and the references to the openwrt builds below will have to go

Reply:
Agreed. Removed openwrt_generic references from the documentation.